tcs-service/src/test/java/com/transformuk/hee/tis/tcs/service/service/mapper/ProgrammeMembershipMapperTest.java

```diff
@@ -54,12 +67,23 @@ public void entityToDtoShouldReturnListOfAllElementsInAsDto() {
     CurriculumMembershipDTO cmDTO3 = new CurriculumMembershipDTO();
     cmDTO3.setId(3L);
 
+    conditionsOfJoining.setVersion(GoldGuideVersion.GG9);
+    conditionsOfJoining.setProgrammeMembershipUuid(pmID);
+    conditionsOfJoining.setSignedAt(Instant.now());
+    ConditionsOfJoiningMapper conditionsOfJoiningMapper = new ConditionsOfJoiningMapperImpl();
+    ConditionsOfJoiningDto conditionsOfJoiningDto
+        = conditionsOfJoiningMapper.toDto(conditionsOfJoining);
```

Comment:
As above, the fields of `conditionsOfJoining` don't matter to the mapper; you could create your own DTO & set the fields on it (or not even set the fields) to remove the ConditionsOfJoiningMapperImpl